code/game/atoms.dm

```diff
@@ -465,8 +466,8 @@ GLOBAL_LIST_EMPTY(blood_splatter_icons)
 /atom/proc/singularity_act()
 	return
 
-/atom/proc/singularity_pull()
-	return
+/atom/proc/singularity_pull(S, current_size)

```

Comment:
obj/singularity/S

Reply:
Can you please make sure all singularity_pull overrides that don't return immediately call ..()

Reply:
hahahahahah LORD HAVE MERCY